.travis.yml

```diff
@@ -1,26 +1,35 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
 sudo: required
+dist: trusty
 services:
   - docker
 env:
   global:
     - PA_TEST_ONLINE_INSTALLER=true
     - PYTHONPATH=$PYTHONPATH:$(pwd)
     - LONG_PRODUCT_TESTS="tests/product/test_server_install.py tests/product/test_status.py tests/product/test_collect.py tests/product/test_connectors.py tests/product/test_control.py tests/product/test_server_uninstall.py"
-  matrix:
-    - OTHER_TESTS=true
-    - SHORT_PRODUCT_TEST_GROUP=0
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN="tests/product/test_server_install.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_status.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_collect.py tests/product/test_server_uninstall.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_connectors.py"
-    - LONG_PRODUCT_TEST_GROUP_PRESTO_ADMIN_AND_PRESTO="tests/product/test_control.py"
+
+matrix:
+  include:
+    - python: "2.7"
```

Comment:
still experimenting with tox on travis.  This only ran py27.  I was worried if I included both it would create two jobs, but we'll see what happens.  I'll let you know when this is ready to be looked at again.